server/src/assembly/resources/conf/logback.xml

```diff
@@ -19,7 +19,7 @@
     under the License.
 
 -->
-<configuration>
+<configuration scan="true" scanPeriod="100 seconds">
```

Comment:
60s is fine. It is a good feature, you'd better point out this change in the commitlog.

Reply:
ok

Reply:
Do not forget to update the related descriptions in docs.